ga4gh/serverconfig.py

```diff
@@ -43,6 +50,9 @@ class ProductionConfig(BaseConfig):
     # We should complain loudly if data source is not set, rather than
     # mysteriously serve no data.
     DATA_SOURCE = None
```

Comment:
I think we should add another config class that extends ProductionConfig called GoogleOidcConfig or something. Our default configs that we already have should continue to work without OIDC. Same for DevelopmentConfig above: we should have an OidcDevelopmentConfig subclass which includes the variable above.